packages/frontend/src/utils/mapUtil.ts

```diff
@@ -0,0 +1,59 @@
+import axios from 'axios';
+
+import { Colors } from '../constants/Colors';
+
+export const loadGeoJson = async (geoJsonPath: string) => {
+  try {
+    const response = await axios.get(geoJsonPath);
+    const contentType = response.headers['content-type'];
+    if (contentType && contentType.indexOf('application/json') !== -1) {
+      const geoJsonData = response.data;
+      return geoJsonData;
+    } else {
+      throw new Error('Invalid content type: ' + contentType);
+    }
+  } catch (error) {
+    console.log('Error loading GeoJSON:', error);
+    throw error;
+  }
+};
+
+export const getPolylineColor = (kpalue: number) => {
```

Comment:
I would keep param and variable names in english if possible.

```suggestion
export const getPolylineColor = (maintenanceArea: number) => {
```